infrastructure/networked_rds/public_keys/prashant.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC9fR9hedRgl1CTAJcjS/r3nkN/fnot1jQNKuL+5JfNoTnjvqXMcx0IX/CChOw1LXj0THbm/CsnQ2aK9lRDvYiR/GC6/dLgHGhbHuFfGVNnRXMvyP+AhgDz7/k6o+N7InDmwPrYLwlV0yUNJDafI/h0SeADz8Z9C/bYEZEW+HM0Y+WLhm5CyRcb8LQbU7ov3NmKweWXGbIYKkMpnI4ftnJbzNnTXktNya9GRgaoFUzIwt+sI0KSlaALcXrmYkoBhReee1+Y+ZROjwDViWtmMJ2BbATfFTm5TzTu8eadSfBREQMAEvtU461mKtAN/25gH8Fa1LcwzcZSBV5iDGRaXuHOF94cYOTLcwNcH70ZgfZV36fYsymihE7yNRJ+7xckt07Ugi2ANA8pHzHBMk4dIwWqjL8D9A6isa16LgZHvil6xVY+52D5KDRnvaitECDm9o64a2s+4ntSVC+D56xxFC7DOyUNwVsVeeDE9EbRUuJ7bjUIC/D9Ot+2bPxAZdXcWnV1NSkPop+EE/6uiOOeWW05b4HNKV5VoqD4f24UFaduKu0d/xVU3xRDFM4Ktd9xzta5qJO8CSBhQ8kNMTB/p3UqEObE5RIUWQG3t/cTeQcdLK3ScPqjg2dHn/NoPaJj1lgAMquAtL5fHzWq0tUZiloiKzONXaeFCq60ozNi5Mmikw== singh.prasha@husky.neu.edu
```

Comment:
this part of the key pair can stay :-) it's public

Reply:
I forgot to mention this folder in .gitignore